### DISCUSSION TITLE
[Unknown Mod][Heroes] Hero receives Morale penalty if he retreats or surrenders from combat

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#3252] [Ideas]
The hero receives a morale penalty if he retreats (flees) or surrenders from a combat.
The penalty is cumulative if he retreats (flees) or surrenders from several combats (there is no limit, it can be -3, -12, -100, etc.).

If the hero retreats (flees) from a combat, a penalty of -2 Morale is applied.
If the hero surrenders from a combat, a penalty of -1 Morale is applied.
If the hero retreats (flees) or surrenders from another combat, an additional penalty of -2 Morale or -1 Morale is applied.

Text is added for the "Current Morale Modifiers:"

When the hero retreats (flees):
"Shame of retreat -2"
Or:
"Shame of retreat -4"
Etc.

When the hero surrenders:
"Cowardice in combat -1"
Or:
"Cowardice in combat -2"
Etc.

However, the final morale taken into account will never be higher than -3 (this is already the case with the other current morale modifiers).
For example, for:

"Current Morale Modifiers:

Troops of 3 alignments -1
Some undead in army -1
Shame of retreat -4
Cowardice in combat -1"

The final morale taken into account will be -3 morale.

The hero recovers 1 point of morale malus for each combat won.
Cowardice in combat is removed first, then Shame of retreat.

This is an option:
"heroes: Hero receives Morale penalty if he retreats or surrenders from a combat"

**Comment by Alucard648** [comment]
I think, that the retreated hero should lose all movement points on buyout.

**Reply by LeHerosInconnu** [reply]
Hello @Alucard648,

> I think, that the retreated hero should lose all movement points on buyout.

I think these are things that should be treated separately.
Also a Morale penalty is more constraining for the hero, compared to a movement penalty that is quickly gone the next day.

**Comment by Alucard648** [comment]
And forgot to menton - morale penalty, coupled with no "1st turn no bad morale trigger" stuff kills hit&run tactic. While this may be good for multiplayer, it renders certain maps that rely on hit&run strategy, unbeatable. Alteris 1 & 2, for example.

**Reply by LeHerosInconnu** [reply]
@Alucard648.

> And forgot to menton - morale penalty, coupled with no "1st turn no bad morale trigger" stuff kills hit&run tactic. While this may be good for multiplayer, it renders certain maps that rely on hit&run strategy, unbeatable. Alteris 1 & 2, for example.

The player will have to make sure that his heroes always have a correct Morale.
There are secondary skills, locations on the adventure map and artifacts for this.
Also the Morale penalty for retreating and fleeing is optional. :)

The hit and run tactic has never been satisfying for me.
And the Alteris and Alteris 2 scenarios can be completed without using it. :)
